src/home_robot/environment.yml

```diff
@@ -7,7 +7,7 @@ dependencies:
   - python==3.9.*
   - cmake
   - pybind11
-  - pinocchio
+  - pin==2.6.17
```

Comment:
This is actually wrong, in conda pinocchio is the correct package

Reply:
Yeah, makes sense. This is fine.